crates/rattler_shell/src/activation.rs

```diff
@@ -13,7 +13,7 @@ use crate::shell::Shell;
 use indexmap::IndexMap;
 use rattler_conda_types::Platform;
 
-const ENV_START_SEPERATOR: &str = "<=== RATTLER ENV START ===>";
```

Comment:
This was doing some weird things - it was executing the `env` executable and completely failing. Although that might also be related to the unicode issues in the first place.

With the new one we don't need any special escaping though.